Do not assume the C++ compiler is g++
Do not fail when $CXX contains a space (e.g. when using ccache)
--- m4/slg_gcc_all_warnings.m4.orig	2014-09-16 13:34:00.000000000 -0500
+++ m4/slg_gcc_all_warnings.m4	2015-09-30 02:07:33.000000000 -0500
@@ -15,12 +15,12 @@
 
 # First, see if we are using CLANG
 using_clang=no
-if (g++ --version 2>&1 | grep clang > /dev/null) ; 
+if ($CXX --version 2>&1 | grep clang > /dev/null) ; 
 then 
-   AC_MSG_NOTICE([g++ is really clang++])
+   AC_MSG_NOTICE([C++ compiler is clang++])
    using_clang=yes
 fi
-if test x$CXX == "xclang++" ; then
+if test "x$CXX" == "xclang++" ; then
    using_clang=yes
 fi
 
